input/utils/fieldset.js

```diff
@@ -64,6 +67,20 @@ Object.defineProperties(Fieldset.prototype, assign({
 		li.appendChild(dom);
 		return li;
 	}),
+	// Allows to retrieve a generated item (field) by path string
+	// (property path must end with provided path string)
+	getItem: d(function (path) {
+		var result = null;
+		path = ensureString(path);
+		if (path[0] === '/') path = path.slice(1);
+		some(this.items, function (item, key) {
+			if ((key === path) || endsWith.call(key, '/' + path)) {
```

Comment:
@medikoo I propose that we do `(key === path)` check before the loop, by simply trying doing this.items[path].